examples/step-30/step-30.cc

```diff
@@ -309,23 +295,19 @@ namespace Step30
 
   // @sect3{Class: DGMethod}
   //
-  // Even the main class of this program stays more or less the same. We omit
-  // one of the assembly routines and use only the second, more effective one
-  // of the two presented in step-12. However, we introduce a new routine
-  // (set_anisotropic_flags) and modify another one (refine_grid).
+  // This declaration is much like that of step-12. However, we introduce a
+  // new routine (set_anisotropic_flags) and modify another one (refine_grid).
   template <int dim>
   class DGMethod
   {
   public:
     DGMethod(const bool anisotropic);
-    ~DGMethod();
 
     void run();
 
   private:
     void setup_system();
-    void assemble_system1();
```

Comment:
Ah, nice. I had noticed a while ago that `assemble_system1()` is declared but never defined here. But then forgot about it again before doing anything about it. Thanks for catching it!